parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileWriter.java

```diff
@@ -69,6 +72,20 @@
   public static final byte[] MAGIC = "PAR1".getBytes(Charset.forName("ASCII"));
   public static final int CURRENT_VERSION = 1;
 
+  // need to supply a buffer size when setting block size. this is the default
+  // for hadoop 1 to present. copying it avoids loading DFSConfigKeys.
+  private static final int DFS_BUFFER_SIZE_DEFAULT = 4096;
```

Comment:
> so this isn't removing an option that users had
Yeah that's what I'm wondering about -- we were using a method before that didn't ask for this info, did that method pull this info from the hadoop configuration? If so -- then users were previously able to change this by messing w/ their hadoop configuration, but now they can't.

On the other hand, if the method we were using before just grabbed the hardcoded default, then this makes sense as it's no different (I'm not worried about duplicating the default, just about whether we've taken away the ability to change this via hadoop configuration)